en/companion_computer/README.md

```diff
@@ -14,14 +14,14 @@ They are connected using a fast serial or Ethernet link, and typically communica
 
 Communications with the ground stations and the cloud are usually routed via the companion computer (e.g. using the [MAVLink Router](https://github.com/mavlink-router/mavlink-router)).
 
-## Integrated Companion/Flight Controller Boards
+## Pixhawk Autopilot Bus Carrier Boards
```

Comment:
This title is wrong. The PAB allows integration with any compatible FC. The point of this section is integration of FC and companion.